ArduCopter/Copter.h

```diff
@@ -820,10 +816,11 @@ class Copter : public AP_Vehicle {
     void motor_test_stop();
 
     // motors.cpp
-    void arm_motors_check();
     void auto_disarm_check();
     void motors_output();
     void lost_vehicle_check();
+    uint32_t auto_disarm_begin;
```

Comment:
normally we don't put the members with the method declarations.. there should be a place lower down where it can fit in.

Reply:
I've moved it and added a comment